build.sbt

```diff
@@ -275,29 +276,32 @@ lazy val docs = http4sProject("docs")
         case _ => Seq.empty
       }
     },
-    includeFilter in makeSite := (
-      "*.html" | "*.css" | 
-      "*.png" | "*.jpg" | "*.gif" | "*.ico" | "*.svg" |
-      "*.js" | "*.swf" | "*.json" | "*.md" |
-      "CNAME" | "_config.yml"
-    ),
-    siteMappings := {
-      if (Http4sGhPages.buildMainSite) siteMappings.value
-      else Seq.empty
-    },
-    siteMappings ++= {
-      val (major, minor) = apiVersion.value
-      for ((f, d) <- tut.value) yield (f, s"docs/$major.$minor/$d")
+    preStageSiteDirectory := sourceDirectory.value / "hugo",
+    siteStageDirectory := target.value / "site-stage",
+    sourceDirectory in Hugo := siteStageDirectory.value,
+    tutSourceDirectory := sourceDirectory.value / "hugo" / "content",
+    tutTargetDirectory := siteStageDirectory.value / "content",
+    copySiteToStage := {
+      streams.value.log.info(s"Copying ${preStageSiteDirectory.value} to ${siteStageDirectory.value}")
+      val src = preStageSiteDirectory.value
+      val dest = siteStageDirectory.value
+      // We skip content because these get tutted
+      val sources = PathFinder(src).*** --- PathFinder(src / "content")
+      val pairs = sources.pair(Path.rebase(src, dest))
+      IO.copy(
+        sources = pairs,
+        overwrite = true,
+        preserveLastModified = true
+      )
     },
+    makeSite <<= makeSite.dependsOn(copySiteToStage, tutQuick),
```

Comment:
Also, `<<=` is deprecated, but I can't make it compile with the suggested `~=`.  I'm showing my sbt ignorance.